src/utils/KML.js

```diff
@@ -171,9 +171,7 @@ const sanitizeFeature = (feature) => {
        */
       image.setRotation(parseFloat(feature.get('iconRotation')) || 0);
 
-      /* Value to be used for icon scaling with map
-       * e.g. icon.setScale(map.getView().getResolutionForZoom(zoomAtMaxIconSize) / map.getView().getResolution())
-       */
+      /* Replaced by pictureOptions, to be removed in react-spatial v2 */
```

Comment:
You can remove this and the next 'if'  , we will do this work in plan-editor

Reply:
Removed all zoomAtMaxIconSize stuff